strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/CleanupExpiredProxyRepositoriesArtifactsCronJobTestIT.java

```diff
@@ -0,0 +1,148 @@
+package org.carlspring.strongbox.cron.jobs;
+
+import org.carlspring.strongbox.config.Maven2LayoutProviderCronTasksTestConfig;
+import org.carlspring.strongbox.domain.ArtifactEntry;
+import org.carlspring.strongbox.event.cron.CronTaskEventTypeEnum;
+import org.carlspring.strongbox.providers.layout.LayoutProvider;
+import org.carlspring.strongbox.providers.repository.ProxyRepositoryProvider;
+import org.carlspring.strongbox.resource.ConfigurationResourceResolver;
+import org.carlspring.strongbox.services.ArtifactEntryService;
+import org.carlspring.strongbox.storage.Storage;
+import org.carlspring.strongbox.storage.repository.Repository;
+
+import javax.inject.Inject;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Optional;
+
+import com.google.common.base.Throwables;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang.time.DateUtils;
+import org.codehaus.plexus.util.StringUtils;
+import org.hamcrest.CoreMatchers;
+import org.hamcrest.Matchers;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import static org.junit.Assert.*;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@ContextConfiguration(classes = Maven2LayoutProviderCronTasksTestConfig.class)
+@RunWith(SpringJUnit4ClassRunner.class)
+public class CleanupExpiredProxyRepositoriesArtifactsCronJobTestIT
```

Comment:
I think that better names for these classes would probably be `CleanupExpiredArtifactsFromProxyRepositoriesCron*`.

Reply:
@fuss86 : I don't think you've applied my remark above,